source/MaterialXContrib/Handlers/TinyEXRImageLoader.cpp

```diff
@@ -5,10 +5,7 @@
 
 #include <MaterialXRender/HardwarePlatform.h>
 
-#if defined(OSWin_) && defined(_WIN64)
-#define TINYEXR_USABLE
-#endif
-#if defined(OSMac_) || defined(OSLinux)
+#if defined(OSWin_) || defined(OSMac_) || defined(OSLinux)
```

Comment:
If all three plattforms are supported now maybe just remove the use of this define?